src/main/java/auto_ckz/site/account/Account.java

```diff
@@ -0,0 +1,70 @@
+package auto_ckz.site.account;
+
+import javax.persistence.*;
+
+import java.time.Instant;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@SuppressWarnings("serial")
+@Entity
+@Table(name = "account")
+public class Account implements java.io.Serializable {
+
+	@Id
+	@GeneratedValue
+	private Long id;
+
+	@Column(unique = true)
+	private String email;
+	
+	@JsonIgnore
+	private String password;
+
+	private String role = "ROLE_USER";
+
+	private Instant created;
+
+    protected Account() {
+
+	}
+	
+	public Account(String email, String password, String role) {
+		this.email = email;
+		this.password = password;
+		this.role = role;
+		this.created = Instant.now();
+	}
+
+	public Long getId() {
+		return id;
+	}
+
+    public String getEmail() {
+		return email;
+	}
```

Comment:
Getters and setters boilerplate can be removed when we will start to use Lombok.